keyserver/handler.go

```diff
@@ -28,18 +28,28 @@ func (server *ConiksServer) handleRegistrationMessage(reg *p.RegistrationRequest
 			p.Error(p.ErrorNameExisted)
 	}
 
-	tb, errCode := server.directory.Register(reg.Username, key)
+	ap, tb, errCode := server.directory.Register(reg.Username, key)
 	if errCode != p.Success {
 		server.Unlock()
 		return p.NewErrorResponse(errCode), p.Error(errCode)
 	}
 	server.tbs[reg.Username] = tb
 	server.Unlock()
 
+	// TODO: do we want to store the user policies into DB?
+
```

Comment:
Yes. It's what I asked you in https://github.com/coniks-sys/coniks-go/pull/29#discussion_r71963914